src/libprojectM/Renderer/Texture.cpp

```diff
@@ -99,7 +99,12 @@ auto Texture::IsUserTexture() const -> bool
     return m_isUserTexture;
 }
 
-void Texture::CreateEmptyTexture()
+auto Texture::Empty() const -> bool
+{
+    return m_textureId == 0;
```

Comment:
I was considering checking the texture id instead of the target. I ended up checking the target, because that was the value that directly triggered the `GL_INVALID_ENUM`. Would it be worth to check both values ?

Reply:
Disregard, just noticed, this is the backport. We should not do anything different here anyways.

Reply:
`m_target` is, except for the volume noise textures, always set to `GL_TEXTURE_2D`, even if the texture is empty. I think "target" is a misleading name for the texture's dimensionality, but it's how OpenGL calls it.

What triggered the error was actually `m_textureId` being zero because the actual texture wasn't initialized at the point when it was first used (copying the previous frame texture into the new preset - naturally, there's no previous texture right after init).